### PR TITLE
Normalizes subsystem naming and definition

### PR DESCRIPTION
**Pull request by Cyberboss**
That's better

### CODE REVIEW

**Comment by YotaXP** [comment]
As much as I like overusing macros, I dunno if this is a good idea.  If you want to split a subsystem into multiple DM files, you're going to have to use `SUBSYSTEM(X)` in one, and the fully qualified path in the other, which seems weird.

**Comment by Cyberboss** [comment]
You'll always have to use the fully qualified if you want a subsystem to do anything other than hold variables. This is just to get rid of having to use NEW_SS_GLOBAL and having to declare the global.

File: code/controllers/subsystem/atoms.dm [hunk 1]
```diff
@@ -1,20 +1,15 @@
-var/datum/controller/subsystem/atoms/SSatoms
-
 #define INITIALIZATION_INSSATOMS 0	//New should not call Initialize
 #define INITIALIZATION_INNEW_MAPLOAD 1	//New should call Initialize(TRUE)
 #define INITIALIZATION_INNEW_REGULAR 2	//New should call Initialize(FALSE)
 
-/datum/controller/subsystem/atoms
+SUBSYSTEM(atoms)
```
Comment (Iamgoofball) [inline code/controllers/subsystem/atoms.dm]: this works???

Comment (RemieRichards) [inline code/controllers/subsystem/atoms.dm]: learn the language you fug.

**Comment by RemieRichards** [comment]
@Cyberboss could call it SUBSYSTEM_DEF, to make it more obvious it's a definition, and not the thing intended to be used everytime (essentially helping Yota's point)

File: code/controllers/subsystem/atoms.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/controllers/subsystem/atoms.dm]: no i code roblox

**Comment by Cyberboss** [comment]
Ree I updated this twice you nerds

**Pull request merged**